Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM node:5
 
 RUN apt-get install libpq-dev
 
-COPY package.json /
+WORKDIR /sequelize
+COPY package.json /sequelize/
 RUN npm install
 
-WORKDIR /sequelize
-VOLUME /sequelize
+COPY . /sequelize/
```

Comment:
@mickhansen Sure, I can revert the changes if you want. But I have to warn you that the `.dockerignore` file only applies to the `COPY` operation and not to the `VOLUME`.
Also, you're performing an `npm install` on the `/` (root) folder and then you mount the volume on the `/sequelize` folder, which means the `npm install` is completely useless.
Anyway, the reason I changed it is because I had some package conflicts due to my node_modules folder being compiled with node v6 (which I use on my workstation) and docker_compose using node v5 (I don't remember exactly which module did not like that and needed to be rebuilt).